body = <<-EOF
<meta name="citation_title" content="GeneNetwork: framework for web-based genetics">
<meta name="citation_author" content="Sloan, Zachary"><meta name="citation_author" content="Arends, Danny"><meta name="citation_author" content="W. Broman, Karl"><meta name="citation_author" content="Centeno, Arthur"><meta name="citation_author" content="Furlotte, Nicholas"><meta name="citation_author" content="Nijveen, Harm"><meta name="citation_author" content="Yan, Lei"><meta name="citation_author" content="Zhou, Xiang"><meta name="citation_author" content="W. Williams, Robert"><meta name="citation_author" content="Prins, Pjotr">
<meta name="citation_publication_date" content="2016/06/17">
<meta name="citation_journal_title" content="The Journal of Open Source Software">
<meta name="citation_pdf_url" content="http://www.theoj.org/joss-papers/joss.00025/10.21105.joss.00025.pdf">
<div class="accepted-paper">
  <h1>GeneNetwork: framework for web-based genetics</h1>
  <div class="columns links">
    <div class="column four-fifths" style="padding-bottom: 10px;">
      <strong>Authors</strong>
      <ul class="author-list">
            <li><a href="http://orcid.org/0000-0002-8099-1363" target="_blank">Zachary Sloan</a></li>
            <li><a href="http://orcid.org/0000-0001-8738-0162" target="_blank">Danny Arends</a></li>
            <li><a href="http://orcid.org/0000-0002-4914-6671" target="_blank">Karl W. Broman</a></li>
            <li><a href="http://orcid.org/0000-0003-3142-2081" target="_blank">Arthur Centeno</a></li>
            <li><a href="http://orcid.org/0000-0002-9096-6276" target="_blank">Nicholas Furlotte</a></li>
            <li><a href="http://orcid.org/0000-0002-9167-4945" target="_blank">Harm Nijveen</a></li>
            <li><a href="http://orcid.org/0000-0001-5259-3379" target="_blank">Lei Yan</a></li>
            <li><a href="http://orcid.org/0000-0002-4331-7599" target="_blank">Xiang Zhou</a></li>
            <li><a href="http://orcid.org/0000-0001-8924-4447" target="_blank">Robert W. Williams</a></li>
            <li><a href="http://orcid.org/0000-0002-8021-9162" target="_blank">Pjotr Prins</a></li>
            </ul>
    </div>
    <div class="one-third column">
      <span class="repo">Repository:<br /><a href="https://github.com/genenetwork/genenetwork2">Repository link &raquo;</a></span>
    </div>
    <div class="one-third column">
      <span class="paper">Paper:<br /><a href="http://www.theoj.org/joss-papers/joss.00025/10.21105.joss.00025.pdf">PDF link &raquo;</a></span>
    </div>
    <div class="one-third column">
      <span class="paper">Review:<br /><a href="https://github.com/openjournals/joss-reviews/issues/25">View review issue &raquo;</a></span>
    </div>

    <div class="one-third column" style="padding-top: 20px;">
      <span class="repo">DOI:<br /><a href="https://doi.org/10.21105/joss.00025">https://doi.org/10.21105/joss.00025</a></span>
    </div>
    <div class="one-third column" style="padding-top: 20px;">
      <span class="paper">Status badge:<br /><img src="http://joss.theoj.org/papers/10.21105/joss.00025/status.svg"></span>
    </div>
    <div class="one-third column" style="padding-top: 20px;">
      <span class="paper">Citation:<br />
      <small>Sloan <em>et al.</em>, (2016), <em>Journal of Open Source Software</em>, 1(2), 25, doi:10.21105/joss.00025</small>
    </div>
  </div>
  <div class="paper-body">
  <h1 id="summary">Summary</h1>
  <p>GeneNetwork (GN) is a free and open source (FOSS) framework for web-based genetics that can be deployed anywhere. GN allows biologists to upload high-throughput experimental data, such as expression data from microarrays and RNA-seq, and also `classic' phenotypes, such as disease phenotypes. These phenotypes can be mapped interactively against genotypes using embedded tools, such as R/QTL <span class="citation">(Arends et al. 2010)</span> mapping, interval mapping for model organisms and pylmm; an implementation of FaST-LMM <span class="citation">(Lippert et al. 2011)</span> which is more suitable for human populations and outbred crosses, such as the mouse diversity outcross. Interactive D3 graphics are included from R/qtlcharts and presentation-ready figures can be generated. Recently we have added functionality for phenotype correlation <span class="citation">(Wang et al. 2016)</span> and network analysis <span class="citation">(Langfelder and Horvath 2008)</span>.</p>
  <div class="figure">
  <img src="qtl2.png" alt="Mouse LMM mapping example" />
  <p class="caption">Mouse LMM mapping example</p>
  </div>
  <p>GN is written in python and javascript and contains a rich set of tools and libraries that can be written in any computer language. A full list of included software can be found in the package named `genenetwork2' and defined in <a href="https://github.com/genenetwork/guix-bioinformatics/blob/master/gn/packages/genenetwork.scm">guix-bioinformatics</a>. To make it easy to install GN locally in a byte reproducible way, including all dependencies and a 2GB MySQL test database (the full database is 160GB and growing), GN is packaged with <a href="https://www.gnu.org/software/guix/">GNU Guix</a>, as described <a href="https://github.com/genenetwork/genenetwork2/blob/master/doc/README.org">here</a>. GNU Guix deployment makes it feasible to deploy and rebrand GN anywhere.</p>
  <h1 id="future-work">Future work</h1>
  <p>More mapping tools will be added, including support for Genome-wide Efficient Mixed Model Association (GEMMA). The <a href="http://www.biodalliance.org/">Biodiallance genome browser</a> is being added as a Google Summer of Code project with special tracks related to QTL mapping and network analysis. Faster LMM solutions are being worked on, including GPU support.</p>
  <p>A REST interface is being added so that data can be uploaded to a server, analysis run remotely on high performance hardware, and results downloaded and used for further analysis. This feature will allow biologist-programmers to use R and Python on their computer and execute computations on GN enabled servers.</p>
  <h1 id="references" class="unnumbered">References</h1>
  <div id="refs" class="references">
  <div id="ref-Arends:2010">
  <p>Arends, D., P. Prins, R. C. Jansen, and K. W. Broman. 2010. &#8220;R/qtl: high-throughput multiple QTL mapping.&#8221; <em>Bioinformatics</em> 26 (23): 2990&#8211;2. doi:<a href="https://doi.org/10.1093/bioinformatics/btq565">10.1093/bioinformatics/btq565</a>.</p>
  </div>
  <div id="ref-WGCNA:2008">
  <p>Langfelder, P., and S. Horvath. 2008. &#8220;WGCNA: an R package for weighted correlation network analysis.&#8221; <em>BMC Bioinformatics</em> 9: 559. doi:<a href="https://doi.org/10.1186/1471-2105-9-559">10.1186/1471-2105-9-559</a>.</p>
  </div>
  <div id="ref-Lippert:2011">
  <p>Lippert, C., J. Listgarten, Y. Liu, C. M. Kadie, R. I. Davidson, and D. Heckerman. 2011. &#8220;FaST linear mixed models for genome-wide association studies.&#8221; <em>Nat Methods</em> 8 (10): 833&#8211;35. doi:<a href="https://doi.org/10.1038/nmeth.1681">10.1038/nmeth.1681</a>.</p>
  </div>
  <div id="ref-Wang:2016">
  <p>Wang, X., A. K. Pandey, M. K. Mulligan, E. G. Williams, K. Mozhui, Z. Li, V. Jovaisaite, et al. 2016. &#8220;Joint mouse-human phenome-wide association to test gene function and disease risk.&#8221; <em>Nat Commun</em> 7: 10464. doi:<a href="https://doi.org/10.1038/ncomms10464">10.1038/ncomms10464</a>.</p>
  </div>
  </div>
  </div>
</div>
EOF
