body = <<-EOF
<meta name="citation_title" content="Correlation Trait Loci (CTL) mapping: phenotype network inference subject to genotype">
<meta name="citation_author" content="Arends, Danny"><meta name="citation_author" content="Li, Yang"><meta name="citation_author" content="A. Brockmann, Gudrun"><meta name="citation_author" content="C. Jansen, Ritsert"><meta name="citation_author" content="W. Williams, Robert"><meta name="citation_author" content="Prins, Pjotr">
<meta name="citation_publication_date" content="2016/10/30">
<meta name="citation_journal_title" content="The Journal of Open Source Software">
<meta name="citation_pdf_url" content="https://github.com/openjournals/joss-papers/blob/master/joss.00087/10.21105.joss.00087.pdf">
<div class="accepted-paper">
  <h1>Correlation Trait Loci (CTL) mapping: phenotype network inference subject to genotype</h1>
  <div class="columns links">
    <div class="column four-fifths" style="padding-bottom: 10px;">
      <strong>Authors</strong>
      <ul class="author-list">
            <li><a href="http://orcid.org/0000-0001-8738-0162" target="_blank">Danny Arends</a></li>
            <li><a href="http://orcid.org/0000-0003-4022-7341" target="_blank">Yang Li</a></li>
            <li><a href="http://orcid.org/0000-0002-4387-2947" target="_blank">Gudrun A. Brockmann</a></li>
            <li><a href="http://orcid.org/0000-0003-2977-9110" target="_blank">Ritsert C. Jansen</a></li>
            <li><a href="http://orcid.org/0000-0001-8924-4447" target="_blank">Robert W. Williams</a></li>
            <li><a href="http://orcid.org/0000-0002-8021-9162" target="_blank">Pjotr Prins</a></li>
            </ul>
    </div>
    <div class="one-third column">
      <span class="repo">Repository:<br /><a href="https://github.com/DannyArends/CTLmapping">Repository link &raquo;</a></span>
    </div>
    <div class="one-third column">
      <span class="paper">Paper:<br /><a href="https://github.com/openjournals/joss-papers/blob/master/joss.00087/10.21105.joss.00087.pdf">PDF link &raquo;</a></span>
    </div>
    <div class="one-third column">
      <span class="paper">Review:<br /><a href="https://github.com/openjournals/joss-reviews/issues/87">View review issue &raquo;</a></span>
    </div>

    <div class="one-third column" style="padding-top: 20px;">
      <span class="repo">DOI:<br /><a href="http://dx.doi.org/10.21105/joss.00087">http://dx.doi.org/10.21105/joss.00087</a></span>
    </div>
    <div class="one-third column" style="padding-top: 20px;">
      <span class="paper">Status badge:<br /><img src="http://joss.theoj.org/papers/10.21105/joss.00087/status.svg"></span>
    </div>
    <div class="one-third column" style="padding-top: 20px;">
      <span class="paper">Cite this paper:<br /><a href="http://www.doi2bib.org/#/doi/10.21105/joss.00087" target="_blank">doi2bib</a>
    </div>
  </div>
  <div class="paper-body">
  <h1 id="summary">Summary</h1>
  <p>The <a href="https://github.com/DannyArends/CTLmapping">CTLmapping repository</a> contains an implementation of the Correlation Trait Loci (CTL) algorithm first presented in <span class="citation">(Danny Arends 2014)</span>. CTL mapping allows geneticists to pursue network inference by discovering genetic loci associated with correlation difference between phenotypes.</p>
  <p>CTL mapping is complementary to the proven quantitative trait locus (QTL) mapping method which maps/associates each separately observed phenotype against genotype. CTL mapping, in contrast, associates correlation differences observed <em>between</em> two phenotypes at a time, subject to the genotype. In other words, QTL mapping treats phenotypes independently while CTL mapping connects phenotypes. CTL show very similar profiles to QTL, but get interesting when they differ (see figure 1).</p>
  <p>CTL differs from mediation [Chick:2016], for example, where the goal is to use covariates tied to genomic position, i.e., mRNA expression, and to find change in strength of QTL signal. The CTL method does not require phenotype tied to genomic location and provides an unbiased method to look for those genomic loci which control correlation differences between phenotypes.</p>
  <p>The CTL method is somewhat related to ANCOVA for QTL mapping which is often used to determine changes in correlation. ANCOVA is used in QTL mapping to adjust for covariates when searching for QTL and is used to improve mapping power, because variance in the dependent variable is absorbed by covariates. CTL, in contrast, are calculated without knowledge of QTL.</p>
  <p>By comparing differences between QTL and CTL and by connecting phenotypes CTL mapping provides a mechanism for inference and discovery of causality (see chapter 4 <span class="citation">(Danny Arends 2014)</span>). This is particularly of interest when phenotype correlations change with conditions, for example in pathways with highly correlated gene expression patterns (see figure 1). CTL mapping differs from existing correlation methods, such as set test methods (e.g., <span class="citation">(K. Wang, Li, and Hakonarson 2010)</span>) in that CTL mapping does not require prior information on sets (e.g., pathways) and uses (existing) QTL information for inference.</p>
  <p>-<img src="https://cloud.githubusercontent.com/assets/4483/19836478/fac4446c-9e77-11e6-93b7-d383aac6b67d.png" width="800px" alt="Figure 1" /> Figure 1: Examples of colocated CTL and QTL profiles, as found in <a href="http://genenetwork.org/">GeneNetwork</a> dataset GN207 (BXD mouse eye mRNA). (a) CTL without a colocalizing QTL between the expression <em>St7</em> and <em>Il18r1</em> genes, i.e., CTL changes at ~ 15 Mb at chromosome 2 from -0.39 B locus, to 0.86 D locus while both genes do not show a difference in mean expression. (b) <em>St7</em> gene shows a QTL at chromosome 6 and no CTL are detected between <em>St7</em> and <em>Il18r1</em> (possibly implying that the expression of this gene is regulated by some variant at this locus). (c) Expression variation of <em>Mtvr2</em> is linked to a CTL with <em>C1qtnf5</em> (0.85 B locus to -0.46 D locus) and to a conventional QTL, both of which map to proximal chromosome 19.</p>
  <p>CTL analysis can be performed on combined phenotypes obtained from the whole spectrum of data types: i.e., from classical phenotypes, such as yield and disease susceptibility, to high-throughput experimental data, such as micro-arrays, RNA-seq and/or protein abundance measurements. This is especially useful in combined datasets, e.g. a combination of: classical phenotypes, protein abundance and gene expression (see figure 2).</p>
  <p>-<img src="https://cloud.githubusercontent.com/assets/4483/19836477/fac3eff8-9e77-11e6-920c-ffca5273874c.png" width="800px" alt="Figure 2" /> Figure 2: By network inference CTL discover the genetic wiring of classical phenotypes and identify known and new key players in the genetic / protein network underlying classical phenotypes using CTL and QTL information; as discovered without prior information in GeneNetwork BXD mouse datasets (again from GN207). Here we visualize significantly changed correlation between genes in different studies as edges between the genes (nodes) with links showing the location (Chr:Position) at which correlation was lost.</p>
  <p>CTL mapping can be applied in model organism experimental crosses, such as mouse and the plant <em>Arabidopsis thaliana</em> (see example datasets below); as well as outbred-crosses, such as the Mouse diversity outbred cross (Mouse-DO), the Rat heterogeneous stock (Rat HS) and in A. <em>thaliana</em> MAGIC <span class="citation">(Koning and McIntyre 2014)</span>; as well as in natural populations, such as human. For statistical power, in general: the more individuals the better. But, as a rule of thumb it is about the same as for QTL, i.e., about 100 individuals for a recombinant inbred line (RIL), and 1,000 individuals for genome-wide association (GWA) in human. This rule of thumb was determined by performing power analysis using genome wide simulations with varying effect sizes and minor allele frequencies. <span class="citation">(Danny Arends 2014)</span>.</p>
  <p>The CTL mapping software is provided as a free and open source (FOSS) package for the R Project for Statistical Computing <span class="citation">(Team 2005)</span>. Data structures of the CTL mapping R package have been harmonized with the popular R/qtl package <span class="citation">(D Arends et al. 2010)</span>, allowing users to quickly and efficiently re-analyze previous (R/)QTL experiments. Additional advantages of close integration with R/qtl are the many input formats supported by R/qtl, and access to all plot and helper functions provided by R/qtl.</p>
  <p>The core CTL mapping algorithm is written in standalone C making it easy to integrate the CTL mapping algorithm into other languages that support bindings to C functions. As a proof of concept the CTL repository provides bindings for the <a href="http://dlang.org/">D programming language</a>.</p>
  <p>CTL has been integrated into <a href="http://genenetwork.org/">GeneNetwork (GN)</a>, a FOSS framework for web-based genetics that can be deployed anywhere <span class="citation">(Sloan et al. 2016)</span>. This allows results from CTL mapping to be interactively explored using the GeneNetwork web interface. Additionally results from CTL mapping can be visualized by plotting routines provided by the R package and results can be exported to external tools (such as Cytoscape <span class="citation">(Shannon et al. 2003)</span>) for visualization and interactive exploration.</p>
  <h1 id="example-datasets">Example datasets</h1>
  <p>CTL mapping comes with several example datasets (in Rdata format) for the user to explore:</p>
  <ul>
  <li>301 gene expression traits measured on 109 <em>Saccharomyces cerevisia</em> <span class="citation">(Brem et al. 2002)</span></li>
  <li>9 Metabolite expression traits measured on 403 <em>Arabidopsis Thaliana</em> <span class="citation">(Blair, Kliebenstein, and Churchill 2012)</span></li>
  <li>24 Metabolite expression traits measured on 162 <em>Arabidopsis Thaliana</em> <span class="citation">(Keurentjes et al. 2006)</span></li>
  </ul>
  <p>(instructions can be found in the <a href="https://github.com/DannyArends/CTLmapping">README</a>).</p>
  <h1 id="future-work">Future work</h1>
  <p>CTL is computationally very intensive, phenotypes <em>O(n<sup>2</sup>)</em>, both in terms of RAM use and CPU. Future work includes research into improving the CTL algorithm for large scale correlations and inference, including the use of GPU/supercomputing. In the context of GeneNetwork we are also working on adding exploratory interactive visualization (such as Cytoscape and D3 interactive graphics).</p>
  <h1 id="references" class="unnumbered">References</h1>
  <div id="refs" class="references">
  <div id="ref-Arends:2010">
  <p>Arends, D, P Prins, R C Jansen, and K W Broman. 2010. &#8220;R/qtl: high-throughput multiple QTL mapping.&#8221; <em>Bioinformatics (Oxford, England)</em> 26 (23). Groningen Bioinformatics Centre, University of Groningen, Groningen, The Netherlands.: 2990&#8211;2. doi:<a href="https://doi.org/10.1093/bioinformatics/btq565">10.1093/bioinformatics/btq565</a>.</p>
  </div>
  <div id="ref-Arends:thesis_chapter">
  <p>Arends, Danny. 2014. &#8220;High-Throughput Computational Methods and Software for Quantitative Trait Locus (QTL) Mapping.&#8221; PhD thesis. <a href="http://hdl.handle.net/11370/29cf3cc5-6596-4d5a-a87c-490b31676a96" class="uri">http://hdl.handle.net/11370/29cf3cc5-6596-4d5a-a87c-490b31676a96</a>.</p>
  </div>
  <div id="ref-Churchill:2012">
  <p>Blair, R H, D J Kliebenstein, and G A Churchill. 2012. &#8220;What can causal networks tell us about metabolic pathways?&#8221; <em>PLoS Computational Biology</em> 8 (4): e1002458. doi:<a href="https://doi.org/10.1371/journal.pcbi.1002458">10.1371/journal.pcbi.1002458</a>.</p>
  </div>
  <div id="ref-Brem:2002">
  <p>Brem, R B, G Yvert, R Clinton, and L Kruglyak. 2002. &#8220;Genetic dissection of transcriptional regulation in budding yeast.&#8221; <em>Science (New York, N.Y.)</em> 296 (5568): 752&#8211;5. doi:<a href="https://doi.org/10.1126/science.1069516">10.1126/science.1069516</a>.</p>
  </div>
  <div id="ref-Keurentjes:2006">
  <p>Keurentjes, J J B, J Fu, R C H de Vos, A Lommen, R D Hall, R J Bino, L H W van der Plas, R C Jansen, D Vreugdenhil, and M Koornneef. 2006. &#8220;The genetics of plant metabolism.&#8221; <em>Nature Genetics</em> 38 (7): 842&#8211;9. doi:<a href="https://doi.org/10.1038/ng1815">10.1038/ng1815</a>.</p>
  </div>
  <div id="ref-DeKoning:2014">
  <p>Koning, D.J. de, and Lauren M McIntyre. 2014. &#8220;GENETICS and G3: Community-Driven Science, Community-Driven Journals.&#8221; <em>G3&amp;#58; Genes|Genomes|Genetics</em> 4 (9). Genetics Society of America: 1567&#8211;8. doi:<a href="https://doi.org/10.1534/g3.114.013680">10.1534/g3.114.013680</a>.</p>
  </div>
  <div id="ref-Cytoscape:2003">
  <p>Shannon, P, A Markiel, O Ozier, N S Baliga, J T Wang, D Ramage, N Amin, B Schwikowski, and T Ideker. 2003. &#8220;Cytoscape: a software environment for integrated models of biomolecular interaction networks.&#8221; <em>Genome Research</em> 13 (11): 2498&#8211;2504. doi:<a href="https://doi.org/10.1101/gr.1239303">10.1101/gr.1239303</a>.</p>
  </div>
  <div id="ref-Sloan:2016">
  <p>Sloan, Zachary, Danny Arends, Karl W. Broman, Arthur Centeno, Nicholas Furlotte, Harm Nijveen, Lei Yan, Xiang Zhou, Robert W. Williams, and Pjotr Prins. 2016. &#8220;GeneNetwork: Framework for Web-Based Genetics.&#8221; <em>JOSS</em> 1 (2). The Open Journal. doi:<a href="https://doi.org/10.21105/joss.00025">10.21105/joss.00025</a>.</p>
  </div>
  <div id="ref-R:2005">
  <p>Team, RDC. 2005. &#8220;R: A language and environment for statistical computing.&#8221; <em>R Foundation for Statistical Computing</em> 1. <a href="http://r-project.kr/sites/default/files/2\%EA\%B0\%95\%EA\%B0\%95\%EC\%A2\%8C\%EC\%86\%8C\%EA\%B0\%9C\_\%EC\%8B\%A0\%EC\%A2\%85\%ED\%99\%94.pdf">http://r-project.kr/sites/default/files/2\%EA\%B0\%95\%EA\%B0\%95\%EC\%A2\%8C\%EC\%86\%8C\%EA\%B0\%9C\_\%EC\%8B\%A0\%EC\%A2\%85\%ED\%99\%94.pdf</a>.</p>
  </div>
  <div id="ref-Wang:2010">
  <p>Wang, K., M. Li, and H. Hakonarson. 2010. &#8220;Analysing biological pathways in genome-wide association studies.&#8221; <em>Nat Rev Genet</em> 11 (12): 843&#8211;54. doi:<a href="https://doi.org/10.1038/nrg2884">10.1038/nrg2884</a>.</p>
  </div>
  </div>
  </div>
</div>
EOF
