body = <<-EOF
<meta name="citation_title" content="VCFPy: a Python 3 library with good support for both reading and writing VCF">
<meta name="citation_author" content="Holtgrewe, Manuel"><meta name="citation_author" content="Beule, Dieter">
<meta name="citation_publication_date" content="2016/10/25">
<meta name="citation_journal_title" content="The Journal of Open Source Software">
<meta name="citation_pdf_url" content="https://github.com/openjournals/joss-papers/blob/master/joss.00085/10.21105.joss.00085.pdf">
<div class="accepted-paper">
  <h1>VCFPy: a Python 3 library with good support for both reading and writing VCF</h1>
  <div class="columns links">
    <div class="column four-fifths" style="padding-bottom: 10px;">
      <strong>Authors</strong>
      <ul class="author-list">
            <li><a href="http://orcid.org/0000-0002-3051-1763" target="_blank">Manuel Holtgrewe</a></li>
            <li><a href="http://orcid.org/0000-0002-3284-0632" target="_blank">Dieter Beule</a></li>
            </ul>
    </div>
    <div class="one-third column">
      <span class="repo">Repository:<br /><a href="https://github.com/bihealth/vcfpy">Repository link &raquo;</a></span>
    </div>
    <div class="one-third column">
      <span class="paper">Paper:<br /><a href="https://github.com/openjournals/joss-papers/blob/master/joss.00085/10.21105.joss.00085.pdf">PDF link &raquo;</a></span>
    </div>
    <div class="one-third column">
      <span class="paper">Review:<br /><a href="https://github.com/openjournals/joss-reviews/issues/85">View review issue &raquo;</a></span>
    </div>

    <div class="one-third column" style="padding-top: 20px;">
      <span class="repo">DOI:<br /><a href="http://dx.doi.org/10.21105/joss.00085">http://dx.doi.org/10.21105/joss.00085</a></span>
    </div>
    <div class="one-third column" style="padding-top: 20px;">
      <span class="paper">Status badge:<br /><img src="http://joss.theoj.org/papers/10.21105/joss.00085/status.svg"></span>
    </div>
    <div class="one-third column" style="padding-top: 20px;">
      <span class="paper">Cite this paper:<br /><a href="http://www.doi2bib.org/#/doi/10.21105/joss.00085" target="_blank">doi2bib</a>
    </div>
  </div>
  <div class="paper-body">
  <h1 id="summary">Summary</h1>
  <p>VCF file format <span class="citation">(Danecek et al. 2011)</span> is the standard file format for genetic variants, both small and structural variants. It has broad adaption in the Bioinformatics community and is used both by most projects, software, and databases these days.</p>
  <p>There is a number of Python libraries for processing VCF, but most focus on reading VCF and not allowing for easily creating or augmenting VCF headers and records. For example, the most popular library PyVCF does not allow for built-in modification of the per-sample <code>FORMAT/*</code> records. PySAM (the wrapper for htslib) does only have very limited support for modifyin VCF records at all.</p>
  <p>VCFPy addresses these issues and provides a well-documented, easy to use, and pythonic interface to reading and writing VCF files. It supports VCF v4.3, reading and writing of both plain-text and bgzip-compressed VCF files, as well as Tabix indices. Further, the project is well-documented and uses automatic testing as well as static code analysis for enforcing software quality standards.</p>
  <h1 id="references" class="unnumbered">References</h1>
  <div id="refs" class="references">
  <div id="ref-Danecek2011">
  <p>Danecek, Petr, Adam Auton, Goncalo Abecasis, Cornelis A Albers, Eric Banks, Mark A DePristo, Robert E Handsaker, et al. 2011. &#8220;The Variant Call Format and VCFtools.&#8221; <em>Bioinformatics</em> 27 (15). Oxford Univ Press: 2156&#8211;8.</p>
  </div>
  </div>
  </div>
</div>
EOF
