body = <<-EOF
<div class="accepted-paper">
  <h1>Xenomapper: Mapping reads in a mixed species context</h1>
  <div class="columns links">
    <div class="column four-fifths" style="padding-bottom: 10px;">
      <strong>Authors</strong>
      <ul class="author-list">
            <li><a href="http://orcid.org/0000-0001-6624-4698" target="_blank">Matthew J. Wakefield</a></li>
            </ul>
    </div>
    <div class="one-third column">
      <span class="repo">Repository:<br /><a href="http://github.com/genomematt/xenomapper/">Repository link &raquo;</a></span>
    </div>
    <div class="one-third column">
      <span class="paper">Paper:<br /><a href="https://github.com/openjournals/joss-papers/blob/master/joss.00018/10.21105.joss.00018.pdf">PDF link &raquo;</a></span>
    </div>
    <div class="one-third column">
      <span class="paper">Review:<br /><a href="https://github.com/openjournals/joss-reviews/issues/18">View review issue &raquo;</a></span>
    </div>

    <div class="one-third column" style="padding-top: 20px;">
      <span class="repo">DOI:<br /><a href="http://dx.doi.org/10.21105/joss.00018">http://dx.doi.org/10.21105/joss.00018</a></span>
    </div>
    <div class="one-third column" style="padding-top: 20px;">
      <span class="paper">Status badge:<br /><img src="http://joss.theoj.org/papers/10.21105/joss.00018/status.svg"></span>
    </div>
    <div class="one-third column" style="padding-top: 20px;">

    </div>
  </div>
  <div class="paper-body">
  <h1 id="summary">Summary</h1>
  <p>Xenomapper is a utility for post processing mapped DNA sequencing reads that have been aligned to a primary genome and a secondary genome, and binning reads into species specific, multimapping in each species, unmapped and unassigned categories. It can be used on single end or paired end sequencing data across a wide range of genomics methods including RNAseq. In paired end data evidence of sequence specificity for either read will be used to assign both reads.</p>
  <p>Use cases include xenografts of human cancers and host pathogen interactions.</p>
  <p>Xenomapper is most effective with mapped reads that include an XS or ZS score that gives the mapping score of the next best read. These include Bowtie2 <span class="citation">(Langmead and Salzberg 2012)</span> and HISAT <span class="citation">(Kim, Langmead, and Salzberg 2015)</span>.</p>
  <p>This work builds upon a similar approach by Rossello et. al. <span class="citation">(Rossello et al. 2013)</span> with a more rigorous implementation and extensions for paired end data and exon aware aligners.</p>
  <h1 id="references" class="unnumbered">References</h1>
  <div id="refs" class="references">
  <div id="ref-HISAT">
  <p>Kim, D., B. Langmead, and S. L. Salzberg. 2015. &#8220;HISAT: A Fast Spliced Aligner with Low Memory Requirements.&#8221; Journal Article. <em>Nat Methods</em> 12 (4): 357&#8211;60. doi:<a href="https://doi.org/10.1038/nmeth.3317">10.1038/nmeth.3317</a>.</p>
  </div>
  <div id="ref-BOWTIE2">
  <p>Langmead, B., and S. L. Salzberg. 2012. &#8220;Fast Gapped-Read Alignment with Bowtie 2.&#8221; Journal Article. <em>Nat Methods</em> 9 (4): 357&#8211;9. doi:<a href="https://doi.org/10.1038/nmeth.1923">10.1038/nmeth.1923</a>.</p>
  </div>
  <div id="ref-ROSSELLO">
  <p>Rossello, F. J., R. W. Tothill, K. Britt, K. D. Marini, J. Falzon, D. M. Thomas, C. D. Peacock, et al. 2013. &#8220;Next-Generation Sequence Analysis of Cancer Xenograft Models.&#8221; Journal Article. <em>PLoS One</em> 8 (9): e74432. doi:<a href="https://doi.org/10.1371/journal.pone.0074432">10.1371/journal.pone.0074432</a>.</p>
  </div>
  </div>
  </div>
</div>
EOF
