body = <<-EOF
<meta name="citation_title" content="biotmle: Targeted Learning for Biomarker Discovery">
<meta name="citation_author" content="S. Hejazi, Nima"><meta name="citation_author" content="Cai, Weixin"><meta name="citation_author" content="E. Hubbard, Alan">
<meta name="citation_publication_date" content="2017/07/26">
<meta name="citation_journal_title" content="The Journal of Open Source Software">
<meta name="citation_pdf_url" content="http://www.theoj.org/joss-papers/joss.00295/10.21105.joss.00295.pdf">
<meta name="citation_doi" content="10.21105/joss.00295">
<meta name="citation_issn" content="2475-9066">
<div class="accepted-paper">
  <h1>biotmle: Targeted Learning for Biomarker Discovery</h1>
  <div class="columns links">
    <div class="column four-fifths" style="padding-bottom: 10px;">
      <strong>Authors</strong>
      <ul class="author-list">
            <li><a href="http://orcid.org/0000-0002-7127-2789" target="_blank">Nima S. Hejazi</a></li>
            <li><a href="http://orcid.org/0000-0003-2680-3066" target="_blank">Weixin Cai</a></li>
            <li><a href="http://orcid.org/0000-0002-3769-0127" target="_blank">Alan E. Hubbard</a></li>
            </ul>
    </div>
    <div class="one-third column">
      <span class="repo">Repository:<br /><a href="https://github.com/nhejazi/biotmle">Repository link &raquo;</a></span>
    </div>
    <div class="one-third column">
      <span class="paper">Paper:<br /><a href="http://www.theoj.org/joss-papers/joss.00295/10.21105.joss.00295.pdf">PDF link &raquo;</a></span>
    </div>
    <div class="one-third column">
      <span class="paper">Review:<br /><a href="https://github.com/openjournals/joss-reviews/issues/295">View review issue &raquo;</a></span>
    </div>

    <div class="one-third column" style="padding-top: 20px;">
      <span class="repo">DOI:<br /><a href="https://doi.org/10.21105/joss.00295">https://doi.org/10.21105/joss.00295</a></span>
    </div>
    <div class="one-third column" style="padding-top: 20px;">
      <span class="paper">Status badge:<br /><img src="http://joss.theoj.org/papers/10.21105/joss.00295/status.svg"></span>
    </div>
    <div class="one-third column" style="padding-top: 20px;">
      <span class="paper">Citation:<br />
      <small>Hejazi <em>et al.</em>, (2017), <em>Journal of Open Source Software</em>, 2(15), 295, doi:10.21105/joss.00295</small>
    </div>
  </div>
  <div class="paper-body">
  <h1 id="summary">Summary</h1>
  <p>The <code>biotmle</code> package provides an implementation of a biomarker discovery methodology based on targeted minimum loss-Based estimation (TMLE) <span class="citation">(van der Laan and Rose 2011)</span> and a generalization of the moderated t-statistic of <span class="citation">(Smyth 2004)</span>, designed for use with biological sequencing data (e.g., microarrays, RNA-seq). The statistical approach made available in this package relies on the use of TMLE to rigorously evaluate the association between a set of potential biomarkers and another variable of interest while adjusting for potential confounding from another set of user-specified covariates. The implementation is in the form of a package for the R language for statistical computing <span class="citation">(R Core Team 2017)</span>.</p>
  <p>There are two principal ways in which the biomarker discovery techniques in the <code>biotmle</code> R package can be used: to evaluate the association between (1) a phenotypic measure (say, environmental exposure) and a biomarker of interest, and (2) an outcome of interest (e.g., survival status at a given time) and a biomarker measurement, both while controlling for background covariates (e.g., BMI, age). By using an estimation procedure based on TMLE, the package produces results based on the Average Treatment Effect (ATE), a statistical parameter with a well-studied causal interpretation (see <span class="citation">van der Laan and Rose (2011)</span> for extended discussions), making the <code>biotmle</code> R package well-suited for applications in bioinformatics, epidemiology, and genomics.</p>
  <p>After adjusting our data set to be consistent with the expect input format -- please consult the vignette accompanying the R package for details -- we would call the principal function of this R package: <code>biomarkertmle</code>.</p>
  <p>We would perform a moderated test on the output of the <code>biomarkertmle</code> function using the function <code>modtest_ic</code>.</p>
  <p>While the principal table of results produced by this R package matches those produced by the well-known <code>limma</code> R package <span class="citation">(Smyth 2005)</span>, there are also several plot methods made available for the <code>bioTMLE</code> S4 class -- subclassed from the popular <code>SummarizedExperiment</code> class -- introduced by this package <span class="citation">(Huber et al. 2015)</span>. For illustrative purposes, we demonstrate the ouput of two such functions on anonymized experimental data below:</p>
  <div class="figure">
  <img src="https://user-images.githubusercontent.com/4483/28638274-61b5e43c-7212-11e7-9480-d226a50dccd0.png" width="800px" alt="Heatmap visualizing the Average Treatment Effect contribution of a change in exposure to each biomarker of interest" />
  <p class="caption">Heatmap visualizing the Average Treatment Effect contribution of a change in exposure to each biomarker of interest</p>
  </div>
  <div class="figure">
  <img src="https://user-images.githubusercontent.com/4483/28638273-61b20bc8-7212-11e7-8c5e-605da83b94f8.png" width="800px" alt="Volcano plot visualizing the log fold change in the Average Treatment Effect against the raw p-value from the moderated t-test performed on each biomarker" />
  <p class="caption">Volcano plot visualizing the log fold change in the Average Treatment Effect against the raw p-value from the moderated t-test performed on each biomarker</p>
  </div>
  
  <h1 id="references" class="unnumbered">References</h1>
  <div id="refs" class="references">
  <div id="ref-huber2015orchestrating">
  <p>Huber, Wolfgang, Vincent J Carey, Robert Gentleman, Simon Anders, Marc Carlson, Benilton S Carvalho, Hector Corrada Bravo, et al. 2015. &#8220;Orchestrating High-Throughput Genomic Analysis with Bioconductor.&#8221; <em>Nature Methods</em> 12 (2). Nature Research: 115&#8211;21.</p>
  </div>
  <div id="ref-R">
  <p>R Core Team. 2017. <em>R: A Language and Environment for Statistical Computing</em>. Vienna, Austria: R Foundation for Statistical Computing. <a href="https://www.R-project.org/" class="uri">https://www.R-project.org/</a>.</p>
  </div>
  <div id="ref-smyth2004linear">
  <p>Smyth, Gordon K. 2004. &#8220;Linear Models and Empirical Bayes Methods for Assessing Differential Expression in Microarray Experiments.&#8221; <em>Statistical Applications in Genetics and Molecular Biology</em> 3 (1): 1&#8211;25.</p>
  </div>
  <div id="ref-smyth2005limma">
  <p>&#8212;&#8212;&#8212;. 2005. &#8220;Limma: Linear Models for Microarray Data.&#8221; In <em>Bioinformatics and Computational Biology Solutions Using R and Bioconductor</em>, 397&#8211;420. Springer.</p>
  </div>
  <div id="ref-vdl2011targeted">
  <p>van der Laan, Mark J, and Sherri Rose. 2011. <em>Targeted Learning: Causal Inference for Observational and Experimental Data</em>. Springer Science &amp; Business Media.</p>
  </div>
  </div>
  </div>
</div>
EOF
